### DISCUSSION TITLE
VolumeGrowthCount - Better to start with only one volume?

### DISCUSSION THREAD

**Discussion by flomo29** [seaweedfs/seaweedfs#2710] [Q&A]
Normaly a collection starts with 7 volumes. But if i have for example only one volume server on a normal harddrive:
Is it better to say volumeGrowthCount = 1, because when i have multiple volumes on one hdd the head from the drive has to hop from file to file having a kind of random write instead of like a sequencle write with only 1 volume file and then a the next volume file after the first is full. What do you think? Multiple volumes makes sense when we have multiple volume servers.

**Comment by chrislusf** [comment]
How to tell that is the bottleneck?

Need to benchmark and see the difference.
